src/app/core/services/auth.service.ts

```diff
@@ -14,7 +14,6 @@ import { DataService } from './data.service';
 import { LabelService } from './label.service';
 import { Title } from '@angular/platform-browser';
 import { GithubEventService } from './githubevent.service';
-import { map } from 'rxjs/operators';
 
 export enum AuthState { 'NotAuthenticated', 'AwaitingAuthentication', 'Authenticated', 'ConfirmOAuthUser'}
```

Comment:
Can we order the states as such: `'NotAuthenticated', 'AwaitingAuthentication', 'ConfirmOAuthUser', 'Authenticated' ?
I think it's clearer to follow the order that the states will take in the "happy path"